cmd/kubemark/hollow-node.go

```diff
@@ -75,25 +85,49 @@ func main() {
 	config.addFlags(pflag.CommandLine)
 	util.InitFlags()
 
-	// create a client for Kubelet to communicate with API server.
+	if !knownMorphs.Has(config.Morph) {
+		glog.Fatal("Unknown morph: %v. Allowed values: %v", config.Morph, knownMorphs.List())
+	}
+
+	// create a client to communicate with API server.
 	cl, err := createClientFromFile(config.KubeconfigPath)
 	if err != nil {
 		glog.Fatal("Failed to create a Client. Exiting.")
 	}
-	cadvisorInterface := new(cadvisor.Fake)
-
-	fakeDockerClient := &dockertools.FakeDockerClient{}
-	fakeDockerClient.VersionInfo = docker.Env{"ApiVersion=1.18"}
-	fakeDockerClient.ContainerMap = make(map[string]*docker.Container)
-	fakeDockerClient.EnableSleep = true
-
-	hollowKubelet := kubemark.NewHollowKubelet(
-		config.NodeName,
-		cl,
-		cadvisorInterface,
-		fakeDockerClient,
-		config.KubeletPort,
-		config.KubeletReadOnlyPort,
-	)
-	hollowKubelet.Run()
+
+	if config.Morph == "kubelet" {
```

Comment:
I think that we should simply run both without asking users to specify any flags.